sys-apps/spu-tools/spu-tools-2.3.0.136.ebuild

```diff
@@ -18,18 +19,20 @@ KEYWORDS="ppc ppc64"
 RDEPEND="
 	sys-libs/ncurses:0=
 "
-DEPEND="
-	${RDEPEND}
+DEPEND="${RDEPEND}"
+BDEPEND="
 	sys-apps/help2man
 	virtual/pkgconfig
 "
 
-S="${WORKDIR}/${PN}/src"
+PATCHES=(
+	"${FILESDIR}"/${P}-buildsystem.patch
+	"${FILESDIR}"/${P}-format-security.patch
+)
 
 src_prepare() {
-	epatch \
-		"${FILESDIR}"/${P}-buildsystem.patch \
-		"${FILESDIR}"/${P}-format-security.patch
+	default
+
 	tc-export CC
```

Comment:
drop `src_prepare`, and do all fo this in `src_configure`

Reply:
done